src/main/java/com/stripe/model/Capability.java

```diff
@@ -154,7 +298,7 @@ public Capability update(CapabilityUpdateParams params, RequestOptions options)
      * verification failed.
      */
     @SerializedName("errors")
-    List<Account.Requirements.Errors> errors;
+    List<Capability.Requirements.Errors> errors;
```

Comment:
That's also a breaking change right? It's correct but you can't change this without breaking people